src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -543,6 +543,9 @@ public GHOrganization getOrganization(String name) throws IOException {
      */
     public GHRepository getRepository(String name) throws IOException {
         String[] tokens = name.split("/");
+        if (tokens.length < 2) {
+            throw new IllegalArgumentException("Repository name must be in format owner/reponame");
```

Comment:
```suggestion
            throw new IllegalArgumentException("Repository name must be in format owner/repo");
```

GitHub docs all use `repo` for this.